src/Neo.VM/Script.cs

```diff
@@ -143,11 +141,12 @@ public Script(ReadOnlyMemory<byte> script, bool strictMode)
         /// <param name="ip">The position to get the <see cref="Instruction"/>.</param>
         /// <returns>The <see cref="Instruction"/> at the specified position.</returns>
         /// <exception cref="ArgumentException">In strict mode, the <see cref="Instruction"/> was not found at the specified position.</exception>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public Instruction GetInstruction(int ip)
         {
-            if (ip >= Length) throw new ArgumentOutOfRangeException(nameof(ip));
             if (!_instructions.TryGetValue(ip, out Instruction? instruction))
             {
+                if (ip >= Length) throw new ArgumentOutOfRangeException(nameof(ip));
```

Comment:
This one is good, the other aggressive seems for big source, are you sure it improve it?

Reply:
they are, but for POC benchs, you are right,,, wont be that much if for normal case~~~